gnd/src/main/java/com/google/android/gnd/persistence/local/room/entity/OptionEntity.java

```diff
@@ -33,14 +32,12 @@
         @ForeignKey(
             entity = FieldEntity.class,
             parentColumns = "id",
-            childColumns = "field_id",
-            onDelete = ForeignKey.CASCADE),
-    indices = {@Index("field_id")})
+            childColumns = "field_id", // NOPMD
```

Comment:
Why no PMD?

Reply:
The literal "field_id" is used 4 times in the same class. So, PMD recommends converting it into a variable.